WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -2453,7 +2455,12 @@ private void CopyChunk()
     {
         try
         {
-            return Data is IRedCloneable irc ? (IRedType)irc.DeepCopy() : Data;
+            return Data switch
+            {
+                IRedBaseHandle baseHandle => baseHandle,
```

Comment:
Here, inside the function called “CopyChunk” we now don’t deepcopy the handle anymore but return a reference to it - even though it is an ICloneable, why?